torch/nn/modules/rnn.py

```diff
@@ -364,10 +365,9 @@ def _replicate_for_data_parallel(self):
 
 
 class RNN(RNNBase):
-    r"""__init__(self,input_size,hidden_size,num_layers=1,nonlinearity='tanh',bias=True,batch_first=False,dropout=0.0,bidirectional=False,device=None,dtype=None)
-
-    Applies a multi-layer Elman RNN with :math:`\tanh` or :math:`\text{ReLU}` non-linearity to an
-    input sequence.
+    r"""Apply a multi-layer Elman RNN with :math:`\tanh` or :math:`\text{ReLU}` non-linearity to an input sequence.
```

Comment:
The signature on the first line is here intentionally as otherwise the sphinx documentation will not render the arguments properly, similarly for LSTM below.

It is okay to ignore the pydocstyle errors for these two and revert these change

You can double check this at the [docs-preview](https://docs-preview.pytorch.org/pytorch/pytorch/113177/index.html) when it renders